### ISSUE TITLE
How to ensure the colors at the same brightness?

### ISSUE THREAD

**Issue by thsno02** [thsno02/potential-giggle#3]
In the same image, the watermarks are at the same brightness even in different colors.

_Originally posted by @thsno02 in https://github.com/thsno02/potential-giggle/issues/1#issuecomment-1216221116_

**Comment by thsno02** [comment]
[Complementary colors](https://en.wikipedia.org/wiki/Complementary_colors): Complementary colors are pairs of [colors](https://en.wikipedia.org/wiki/Color) which, when combined or [mixed](https://en.wikipedia.org/wiki/Color_mixing), cancel each other out (lose [hue](https://en.wikipedia.org/wiki/Hue)) by producing a [grayscale](https://en.wikipedia.org/wiki/Grayscale) color like [white](https://en.wikipedia.org/wiki/White) or [black](https://en.wikipedia.org/wiki/Black).

In other words, the sum of the RGB value is equal to 255 respectively.
